etcdctl/ctlv2/command/util.go

```diff
@@ -78,7 +79,7 @@ func getPeersFlagValue(c *cli.Context) []string {
 
 	// If we still don't have peers, use a default
 	if peerstr == "" {
-		peerstr = "http://127.0.0.1:2379,http://127.0.0.1:4001"
```

Comment:
probably better to avoid arch specific paths if possible...

is `ETCDCTL_API=3 etcdctl` also affected?